source/code-snippets/usage-examples/deleteOne.js

```diff
@@ -1,26 +1,37 @@
+/* Delete a document */
+
+// Import the MongoClient type from the mongodb package.
 import { MongoClient } from "mongodb";
 
 // Replace the uri string with your MongoDB deployment's connection string.
 const uri = "<connection string uri>";
 
+// Create a new client and connect to MongoDB.
 const client = new MongoClient(uri);
 
 async function run() {
   try {
+    // Access the movies collection from the sample_mflix database.
     const database = client.db("sample_mflix");
     const movies = database.collection("movies");
 
-    // Query for a movie that has title "Annie Hall"
+    // Create a filter for movies with a title that is "Annie Hall".
     const query = { title: "Annie Hall" };
 
+    // Delete the first document that matches the preceding query filter.
     const result = await movies.deleteOne(query);
+
+    // Print the result of the operation, depending on if a document was
+    // deleted or not.
```

Comment:
Issue:

From the [style guide entry "if"](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#h):
> Use if to introduce an adverbial clause that describes a condition on which an action depends.

Suggestion:
"// Print a message that indicates whether the operation deleted a document."